actix/src/handler.rs

```diff
@@ -1,9 +1,6 @@
-use std::fmt;
-use std::future::Future;
-use std::pin::Pin;
-use std::sync::Arc;
+use std::{fmt, future::Future, pin::Pin, sync::Arc};
 
-use tokio::sync::oneshot::Sender as SyncSender;
+pub use tokio::sync::oneshot::Sender as OneshotSender;
```

Comment:
Why is this a `pub use`?

Reply:
actix-derive have macro for impl MessageResponse trait.

Reply:
doc hidden?

Reply:
The derive macro would import it from actix::dev module. hide it would make people look into the expand code and wonder what is that.

Reply:
I think re-export the channel stuff in actix-rt could be a better way to hide all these tokio types.